### ISSUE TITLE
TensorRT model

### ISSUE THREAD

**Issue by Cat11438** [Zhen-Dong/HAWQ#16]
Does HAWQ supports conversion to TensorRT models? Thanks

**Comment by zachzzc** [comment]
No, we don't have a tensorRT model now. Since we separate the training part in Pytorch and inference part in TVM, it is possible to build a tensorRT model to read the trained parameters and use it. We would really appreciate it if you want to try it out.
